molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CommandLineAnnotatorConfig.java

```diff
@@ -43,6 +45,12 @@ AnnotationService annotationService()
 		return new AnnotationServiceImpl();
 	}
 
+	@Bean
+	JarRunner jarRunner()
+	{
+		return new JarRunnerImpl();
+	}
+
```

Comment:
This one should sit in the shared config, not in the commandline-only config. Now the application no longer starts up.